app/controllers/reports_controller.rb

```diff
@@ -11,6 +11,30 @@ def call_volumes
     end
   end
 
+  def satisfaction_summary
+    @satisfactions = Satisfactions.new(satisfaction_params)
+    @satisfaction_summary = SatisfactionSummary.new(@satisfactions.results)
+
+    respond_to do |format|
+      format.html
+      format.csv do
+        filename = "satisfaction_#{@satisfactions.period}"
```

Comment:
Has there been an expressed requirement to include the period and stuff in the exported filenames? It adds a bit of noise and if it isn't particularly helpful to the data folks then we could simplify this a bit.